scripts/lib/setup-apt-repo

```diff
@@ -42,6 +35,18 @@ fi
 if [ "$zulip_source_hash" = "$(sha1sum "$SOURCES_FILE")" ] && ! [ -e "$STAMP_FILE" ]; then
     echo "zulip.list file did not change; skipping apt-get update"
 else
+    if [ "$release" = "trusty" ] || [ "$release" = "xenial" ] || [ "$release" = "bionic" ]; then
+        apt-get install -y lsb-release apt-transport-https gnupg
+        apt-key add "$SCRIPTS_PATH"/setup/pgroonga-ppa.asc
+        apt-key add "$SCRIPTS_PATH"/setup/zulip-ppa.asc
+    elif [ "$release" = "stretch" ]; then
+        apt-get install -y lsb-release apt-transport-https gnupg debian-archive-keyring
```

Comment:
This is so that `apt-get` is run only once. Previously it was `apt-get install -y lsb-release apt-transport-https gnupg` and `apt-get install -y debian-archive-keyring`.